internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -0,0 +1,431 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package quickstart
+
+import (
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"math/big"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/convert"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/net"
+	"github.com/mongodb/mongocli/internal/randgen"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const quickstartTemplate = "Now you can connect to your Atlas cluster with: mongo -u %s -p %s %s\n"
+
+const (
+	replicaSet        = "REPLICASET"
+	diskSizeGB        = 20.0
+	mdbVersion        = "4.4"
+	shards            = 1
+	tier              = "M20"
+	members           = 3
+	zoneName          = "Zone 1"
+	accessListComment = "IP added with mongocli atlas quickstart"
+	atlasAdmin        = "atlasAdmin"
+	none              = "NONE"
+	passwordLength    = 12
+	max               = 1000000
+)
+
+// DefaultRegions represents the regions available for each cloud service provider
+var DefaultRegions = map[string][]string{
+	"AWS":   {"US_EAST_1", "US_WEST_2", "AP_SOUTH_1", "AP_EAST_2", "EU_WEST_1", "EU_CENTRAL_1", "ME_SOUTH_1", "AF_SOUTH_1"},
+	"GCP":   {"CENTRAL_US", "CANADA_CENTRAL", "WESTERN_EUROPE", "ASIA_SOUTH_EAST", "SOUTH_AFRICA_NORTH", "UAE_NORTH"},
+	"AZURE": {"US_EAST_2", "US_WEST", "EUROPE_NORTH"},
+}
+
+type Opts struct {
+	cli.GlobalOpts
+	cli.WatchOpts
+	cli.ListOpts
```

Comment:
[nit] the user is not paginating so this is kind of a hacky way to do what I think you're doing